src/components/common/workspace/AddWorkspace.tsx

```diff
@@ -1,12 +1,69 @@
-import { AddWorkspaceContainer } from './Container';
+import styled from '@emotion/styled';
 import plusLogo from '../../../resources/image/plusLogo.png';
-const AddWorkspace = (props: any) => {
+import { Workspace } from '@@types/index';
+import useAdminUser from '@hooks/useAdminUser';
+import { useRef, useState } from 'react';
+import InputModal from '../modal/InputModal';
+import { AddWorkspaceModalContent } from '../content/AddworkspaceModalContent';
+
+const AddWorkspaceContainer = styled.form`
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 321px;
+  height: 332px;
+  border-radius: 25px;
+  border: 1px solid #000;
+`;
+
+type WrapperProps = {
+  workspaces: Workspace[];
+};
+
+const AddWorkspace = ({ workspaces }: WrapperProps) => {
+  const maxWorkspaceNum = 3;
```

Comment:
```suggestion
  const maxWorkspaceNumber = 3;
```

줄인다면 num으로 줄이는 것보다는 no로 더 많이 줄이는 편이긴 합니다